integr8ly/integreatly-operator
# MGDAPI-5874 Addition of PO resource stability e2e

# Issue link
[MGDAPI-5874](https://issues.redhat.com/browse/MGDAPI-5874)

# What
The addition of a new e2e test for resource stability, this tests checks that resources arent being continuously updated.  Threescale metrics monitor was used in this case.

# Verification steps
1. Checkout this branch 
2.  Install RHOAM
3.  Verify test passes by running 
```
INSTALLATION_TYPE=managed-api TEST="Verify package operator" make test/e2e/single
```
4. Verify test fails by running the command below, but when the `waiting 30 seconds` log shows update the `threescale-operator-controller-manager-metrics-monitor` in the `redhat-rhoam-3scale-operator` namespace. The test should fail
```
INSTALLATION_TYPE=managed-api TEST="Verify package operator" make test/e2e/single
```


Alterntively you can install RHOAM via OLM by using this catalogSource
```
oc apply -f -<<EOF
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: rhmi-operators
  namespace: openshift-marketplace
spec:
  sourceType: grpc
  image: quay.io/pstefans/managed-api-service-index:1.39.0

EOF
```
